packages/scripts/__tests__/migrate-glintrc.test.ts

```diff
@@ -0,0 +1,475 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { fileURLToPath } from 'url';
+
+import { parse } from 'json5';
+import { afterEach, beforeEach, describe, test, expect } from 'vitest';
```

Comment:
Actual imports? 🤯 This is the *real* win over Jest. 😏 I can actually report, though: vitest is good.

Reply:
Neat! I'll have to look at moving the rest of our stuff over (I _suspect_ vitest won't require as many gymnastic feats of module trickery to get running in the Code testing environment)